### PR TITLE
Esy manifest + CI setup

### PR DESCRIPTION
**Pull request by ManasJayanth**
This package does not have `--disable-cfi`

### CODE REVIEW

**Comment by ManasJayanth** [comment]
cc @EduardoRFS

File: .gitattributes [hunk 1]
```diff
@@ -148,6 +148,19 @@ build-aux/config.guess text eol=lf
...
+config/auto-aux/searchpath text eol=lf
+config/auto-aux/solaris-ld text eol=lf
+config/auto-aux/tryassemble text eol=lf
+config/auto-aux/trycompile text eol=lf
+config/gnu/config.guess text eol=lf
+config/gnu/config.sub text eol=lf
+esy-build text eol=lf
+esy-configure text eol=lf
+clone-flexdll text eol=lf
+configure-windows text eol=lf
```
Comment (EduardoRFS) [inline .gitattributes:163]: really nice, I had problems before regarding windows and this

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ocaml",
+  "version": "4.10.2000",
+  "description": "OCaml Compiler as an esy npm Package",
+  "versionNotes": [
+    "See the README for notes about how the version of this package corresponds to the official OCaml compiler version."
+  ],
+  "esy": {
+    "build": [
+      "./esy-configure --prefix $cur__install",
```
Comment (EduardoRFS) [inline package.json:10]: \o\

**Review by EduardoRFS — APPROVED** [review]
Amazing job, great to see the CI testing for code
anime girl happy about this PR

**Comment by tatchi** [comment]
Here's the error I got on windows:

unknown

**Pull request merged**